app/views/publishers/promo_activate.html.slim

```diff
@@ -0,0 +1,14 @@
+- content_for(:navbar_content) do
```

Comment:
These `content_for(:navbar_content)` blocks in all these files have no content. Can they simply be removed?

Reply:
Yes they can; I've removed them.  Thanks for the comment.